tests/sqllogictests/suites/base/03_dml/03_0025_delete_from

```diff
@@ -119,13 +119,13 @@ select count(*) = 0 from t
 statement ok
 insert into t values (1),(2),(NULL)
 
-statement ok
+statement error 1001
 delete from t where now()
```

Comment:
Is this case related to the number-to-bool rule for `is_try`?

Reply:
I will add is_deterministic check for deletion.